ledger/participant-integration-api/src/main/scala/platform/apiserver/error/LedgerApiErrors.scala

```diff
@@ -402,6 +402,27 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
 
       }
 
+      @Explanation(
+        """TODO: fill it in"""
+      )
+      @Resolution("""TODO: fill it in""")
+      object LedgerConfigurationNotFound
```

Comment:
```suggestion
      @Explanation(
        """The ledger configuration could not be retrieved. This could happen due to incomplete initialization of the participant or due to an internal system error"""
      )
      @Resolution("""Ask the operator to restart the participant or contact support.""")
      object LedgerConfigurationNotFound
```
But @meiersi-da could comment better on this.

Reply:
Thanks @tudor-da !

Reply:
LGTM @tudor-da